WRITING_DOCS.md

```diff
@@ -9,8 +9,6 @@
   - [Divider](#divider)
```

Comment:
Also, when you hit a 404, it still highlights a category, when it shouldn't.
You can see in the above screenshot that "Slice" is highlighted on the new site,
even though the 404 page isn't in the Slice category.